### ISSUE TITLE
⚠️ Web SQL is deprecated

### ISSUE THREAD

**Issue by tomayac** [mWater/minimongo#83]
Web SQL is on its way out and is being [slowly deprecated](https://developer.chrome.com/blog/deprecating-web-sql/). Web SQL access in insecure contexts is deprecated as of Chromium 105 at which time a warning message will be shown in the Chrome DevTools Issue panel. It's planned to fully remove it in later versions. This is a heads up to consider removing Web SQL as a storage backend from your library.

**Comment by grassick** [comment]
True, however it's the primary storage in Cordova, so we still need to use it for that, as Cordova is the primary platform that we use this library for.
